STYLE.md

```diff
@@ -0,0 +1,48 @@
+# Style Guide
+
+This guide contains general guidelines for how to format and present documentation in this repository.
+They should be followed for most cases, but as a guideline it can be broken, _with good reason_.
+
+The documentation is written using [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/).
+
+## General Formatting
+
+- Format paragraphs with one sentence per line for easier diffs.
+- Leave a space before and after headings.
+
+## Headings are Written in Title Case
+
+Use [title case](https://en.wikipedia.org/wiki/Letter_case#Title_case) for headings, meaning all words are capitalized except for minor words.
+
+## Avoid Nesting Headings too Deep
+
+Nesting headings up to three levels is generally ok.
+
+## Unordered and Ordered Lists
+
+Write lists as proper sentences.
+Separate the items simply with commas if each item is simple, or make each item a full sentence if the items are longer and contain multiple sentences.
+
+1. The first item can look like this,
+2. The second like this, and
+3. The third item like this.
+
+## Using Admonitions
+
+Aim to include examples, notes, warnings using [admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) whenever appropriate.
+They stand out better from the main text, and can be collapsed by default if needed.
+
+!!! example "Example one"
```

Comment:
Some admonition titles use all lower case, some sentence case. Preference?

Reply:
Of course not a big deal. This is just aiming to bring a bit of consistency, but we'll break the rules anyway, intentionally or unintentionally.